pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "up42-py"
-version = "0.37.0a5"
+version = "0.37.0a6"
```

Comment:
To be updated

Reply:
I am a bit confused.
Which version should I update it to?

Reply:
I am a bit confused.
Which version should I update it to?

Reply:
Sorry, unmerged PRs confusion 😄